kaali.js

```diff
@@ -0,0 +1,1371 @@
+// Copy Script Link
+content = document.querySelector("#script-link");
+if (content != null) {
+  content.addEventListener("click", () => {
+    navigator.clipboard.writeText(
+      '<script src="https://kaaliui.netlify.app/kaali.js"></>'
+    );
+
+    var tooltip = document.querySelector("#scripttooltip");
+
+    tooltip.innerHTML = "Copied";
+  });
+  content.addEventListener("mouseout", () => {
+    var tooltip = document.querySelector("#scripttooltip");
+    tooltip.innerHTML = "Copy to Clipboard";
+  });
+}
```

Comment:
This should be changed to a function whcih accepts the tooltipID and writeText as a param.

Reply:
Changed to a function which accepts the tooltipID and writeText as a param.

Reply:
changed to a function which accepts the tooltipID and writeText as a param.